10_datastructures_lab/02.py

```diff
@@ -1,5 +1,20 @@
-"""
-Write a program that prints only unique values
-from sys.argv (without program name)
-"""
-
+"""
+Looks for anagrams and print the couples
+"""
+from collections import Counter
+
+src = "C:\Users\lzeno\OneDrive\Code\Python Scripts\ToCode Course\Tasks\Data Structures\Anagram.txt"
+
+lines = open(src,"r").readlines()
+counters =[]
+
+for i in range(len(lines)):
+    lines[i] = lines[i].replace('\n','')
+

```

Comment:
It'll be shorter (and with the same result) to: `lines = open(src, 'r').read().splitlines()`